source/mobile/backend/remote-config-runbook.html.md

```diff
@@ -52,6 +52,14 @@ Here, you can see that the default configuration is used in all environments - p
 
 Below are instructions for some common tasks that may be needed to fix urgent situations.
 
+## Pre-requisites
+
+> In an emergency, if you are struggling to install the pre-requisites, you could skip the local validation and generation steps and allow the CI pipeline to do this. This is not recommended, however.
+
+Node.js (and `npm`) must be installed. General intructions are [here](https://nodejs.org/en/learn/getting-started/how-to-install-nodejs) although a popular approach is to use [nvm](https://github.com/nvm-sh/nvm).
```

Comment:
What you have is fine (although you've a typo on line 59 `intructions` instead of `instructions`.

More verbose attempt:

```
Node.js, npm and TypeScript must all be installed. 

You can check if you have them available by running 
`node -v` or `npm -v` or `tsc -v` 

General installation instructions for node.js are [here](https://nodejs.org/en/learn/getting-started/how-to-install-nodejs) although a popular approach is to use [nvm](https://github.com/nvm-sh/nvm).

The installation of TypeScript can then normally be done with `npm install typescript`

```

Reply:
Yeah that's far better - thank you